apps/backend/src/util/request-location.ts

```diff
@@ -0,0 +1,22 @@
+import type { Request } from "express";
+
+export interface RequestLocation {
+  city: string;
+  country: string;
+}
+
+/**
+ * Extract location information from request headers.
+ */
+export function extractLocationFromRequest(
+  req: Request,
+): RequestLocation | null {
+  const city = req.headers["cf-ipcity"];
+  const country = req.headers["cf-ipcountry"];
```

Comment:
Why do you request the location? Is it for security purpose?

Reply:
Yes but it's present in the request because we use Cloudflare proxy.